### DISCUSSION TITLE
Missing Validation Code in Documenation

### DISCUSSION THREAD

**Discussion by nCubed** [keygen-sh/community#34] [Issue]
When using the `validate` method on a non-existent key, the `meta.code` value returned is not included in the documentation.

**Response Json**
```json
{
    "data": null,
    "meta": {
        "ts": "2022-06-02T19:50:19.963Z",
        "valid": false,
        "detail": "does not exist",
        "constant": "NOT_FOUND",
        "scope": {
            "product": "f49833ec-<snip>",
            "policy": "5a8af2f4-<snip>",
            "fingerprint": "<snip>"
        }
    }
}
```

The [Validation Codes documentation](https://keygen.sh/docs/api/licenses/#licenses-licenses-actions-validate-response-codes-information) is missing the `NOT_FOUND` constant.

Not sure if this is the correct "category" to report this information. If there's a better one, please let me know. Thanks!

**Comment by ezekg** [comment]
You're looking at the docs for the [validate by ID](https://keygen.sh/docs/api/licenses/#licenses-actions-validate) action, which does not return a `NOT_FOUND` validation code, rather it will return a 404 error response if the license does not exist. You're likely looking for the docs on the [validate by key](https://keygen.sh/docs/api/licenses/#licenses-actions-validate-key) action, which does return a `NOT_FOUND` validation code when the license key does not exist, since it doesn't perform a license lookup beforehand.

Hope that makes sense? Let me know!

**Reply by nCubed** [reply]
Doh! Now I need to go update my errors 🤦🏼‍♀️  Just a hold-over from when I was using `byId` and now using `byKey`.
